c10/util/complex.h

```diff
@@ -144,10 +144,10 @@ namespace c10 {
 //  - thrust::complex only support float and double
 
 template <typename T>
-struct alignas(sizeof(T) * 2) complex {
```

Comment:
I'm not a C++ foo, but since this structure is already of size `sizeof(T) * 2`, what does this alignment request do?
Isn't it a noop anyways?